discord/utils.py

```diff
@@ -78,6 +79,17 @@ def decorator(func):
         return CachedSlotProperty(name, func)
     return decorator
 
+class SequenceProxy(collections.abc.Sequence):
```

Comment:
This is missing the following to satisfy the interface:

1. `__iter__`
2. `__reversed__`
3. `__contains__`

Reply:
Those are implemented by the abc